README.md

```diff
@@ -1,23 +1,50 @@
-<h1 align="center"> ReactNativeNotes </h1>
+<p align="center">
+  <img src="./Resources/img/RNN_README_logo_icon_orange.png"/>
+  <h1 align="center"> ReactNativeNotes </h1>
+</p>
 <p align="center">
     Demo application to create and keep your notes and tasks.
 </p>
+<p align="center">
+    <a href="https://github.com/callstack/ReactNativeNotes/blob/main/LICENSE">
+        <img src="https://img.shields.io/github/license/callstack/ReactNativeNotes?style=plastic" alt="ReactNativeNotes is released under the MIT license." />
+    </a>
+    <a href="https://app.circleci.com/pipelines/github/callstack/ReactNativeNotes">
+        <img src="https://circleci.com/gh/callstack/ReactNativeNotes/tree/main.svg?style=shield" alt="Build on CircleCI" />
+    </a>
+</p>
+
+
+This project is a demonstration application that presents abilites and skills of [Callstack](https://callstack.com) company in creating React Native Windows **desktop** applications.
```

Comment:
```suggestion
This project explores possibilities of React Native Windows as a framework for building desktop apps.
```

Reply:
Looks better and is shorter, I'll take that.
Thank you 👍